via/pshell.py

```diff
@@ -1,2 +1,21 @@
-def setup(_env):  # pragma: no cover
-    return
+from via import models
+from tests import factories
+from tests.factories.factoryboy_sqlalchemy_session import (
+    set_factoryboy_sqlalchemy_session,
+)
+
+
+def setup(env):
+    env["tm"] = env["request"].tm
+    env["tm"].__doc__ = "Active transaction manager (a transaction is already begun)."
+    env["request"].tm.begin()
+
+    env["db"] = env["request"].db
+    env["db"].__doc__ = "Active DB session."
+
+    env["m"] = env["models"] = models
+    env["m"].__doc__ = "The via.models package."
+
+    env["f"] = env["factories"] = factories
+    env["f"].__doc__ = "The test factories for quickly creating objects."
+    set_factoryboy_sqlalchemy_session(env["request"].db)
```

Comment:
This just makes the models, DB session and factories convenient to use from `make shell`. See the testing instructions in https://github.com/hypothesis/via/pull/1072 for an example